src/utils/clients.ts

```diff
@@ -63,12 +62,13 @@ const setDeviceState = (() => {
 			}
 
 			const eventType = targetState.connected ? 'connect' : 'disconnect';
-			return pooledPostAsync({
-				url: `https://${BALENA_API_HOST}/services/vpn/client-${eventType}`,
-				timeout: REQUEST_TIMEOUT,
-				form: _.extend({ service_id: service.getId() }, targetState),
-				headers: { Authorization: `Bearer ${apiKey}` },
-			})
+			return request
+				.post({
+					url: `https://${BALENA_API_HOST}/services/vpn/client-${eventType}`,
+					timeout: REQUEST_TIMEOUT,
+					form: _.extend({ service_id: serviceId }, targetState),
```

Comment:
Why is this `form` and not `json`?

Reply:
Good question, looks like it has always been that way. I'll update it.